cgi-bin/LJ/HTMLControls.pm

```diff
@@ -439,27 +422,14 @@ sub html_color {
             'maxlength' => 7,
             'name'      => $htmlname,
             'id'        => $htmlname,
-            'onchange'  => "setBGColorWithId(findel('${htmlname}_disp'),'${htmlname}');",
             'onfocus'   => $opts->{'onchange'},
             'disabled'  => $opts->{'disabled'},
             'value'     => $opts->{'default'},
             'noescape'  => 1,
-            'raw'       => $opts->{'raw'},
+            'raw'       => $opts->{'raw'} . "data-coloris",
```

Comment:
If there is raw here, will there be a trailing space or do we need to add one? Or is it okay if whatever is in raw is smashed up with `data-coloris`?

Reply:
Oof good catch. I don't think the places we call the color picker right now pass in any raw arguments buuuut I'd rather be safe for the future.